package.json

```diff
@@ -50,6 +50,7 @@
   "types": "dist/index.d.ts",
   "pkg": {
     "assets": [
+      "node_modules/jose/**/*",
```

Comment:
We don't know why `jose` is needed. But you won't be able to tell until you actually run the final program. I recommend leaving it here until we verify why `jose` as an asset is required.

Reply:
I can't find any native addons or JSON files except for `package.json` in jose.

Reply:
This will require a manual check until we have integration testing for the built binaries.